libp2p/Host.cpp

```diff
@@ -155,6 +155,9 @@ void Host::stop()
 	// stop worker thread
 	if (isWorking())
 		stopWorking();
+
+	// This destroyes EthereumHost objects.  EthereumHost objects contain a const reference to Client's, so they need to disappear first.
```

Comment:
What did you mean by this? I think `EtherumHost` has references to member of `Client`, but not to `Client` itself?

Reply:
Updated the comment.